lib/src/layout/layout.dart

```diff
@@ -69,33 +76,177 @@ class _AgoraVideoViewerState extends State<AgoraVideoViewer> {
     widget.client.sessionController.dispose();
   }
 
-  @override
-  Widget build(BuildContext context) {
+  Widget userLayout(String? layoutType, String? baseUrl) {
     return GestureDetector(
       behavior: HitTestBehavior.translucent,
-      child: widget.layoutType == Layout.floating
-          ? FloatingLayout(
-              client: widget.client,
-              disabledVideoWidget: widget.disabledVideoWidget,
-              floatingLayoutContainerHeight:
-                  widget.floatingLayoutContainerHeight,
-              floatingLayoutContainerWidth: widget.floatingLayoutContainerWidth,
-              floatingLayoutMainViewPadding:
-                  widget.floatingLayoutMainViewPadding,
-              floatingLayoutSubViewPadding: widget.floatingLayoutSubViewPadding,
-              showAVState: widget.showAVState,
-              showNumberOfUsers: widget.showNumberOfUsers,
-              videoRenderMode: widget.videoRenderMode,
+      child: layoutType == "floating"
```

Comment:
Is there an easy way to make this not compare to a string in this way? instead an enum with a raw value (that may just be a swift behaviour, not available in Dart)